### ISSUE TITLE
osu cleaner runs as a background process 

### ISSUE THREAD

**Issue by zyuaboggy** [TcNobo/TcNo-osu-Cleaner#5]
![image](https://user-images.githubusercontent.com/88302841/127803454-5c8c3586-62b6-4efb-a43c-1aca9fbda927.png)
when opening the exe, it runs in the background instead of opening the program... any way to fix this? 
I have Microsoft .NET Framework 4.5.2 Runtime and I also tried running it as an administrator. 
![image](https://user-images.githubusercontent.com/88302841/127804204-236af2b5-4dbb-4849-bcd6-ec7f921681f5.png)
if anyone could help me out that would be great! I'll also see if I can post a solution for others if I find one and if this is an actual issue and not just me making a dumb mistake.

**Comment by CrypticXT** [comment]
Got the same issue. Let it run in background and after 1h the tool finally opened a window but it wasn't reacting

**Comment by radiume** [comment]
I've got the same issue.. is there a fix to it yet?

**Comment by CrypticXT** [comment]
> I've got the same issue.. is there a fix to it yet?

Just use the previous version

**Comment by navidmafi** [comment]
> 
> 
> Got the same issue. Let it run in background and after 1h the tool finally opened a window but it wasn't reacting

It looks like a thread-locking problem

**Comment by TcNobo** [comment]
Can't believe I missed something so simple... The program checks for more than 1 account and will list usernames for you to pick from. If there's one file, it'll only use that one available username.
This is in a loop because if a user closes the dialog, it needs to reopen as a username MUST be selected.
For some reason the single file and multiple file paths are both in an infinite loop (waiting for a user to pick a file), but a user was never shown that window if there was 1 account present.

TLDR: This has been fixed, and will be available in another build soon. Working on a new feature as well :)
The missing break is added below:
![devenv_J1Oj6q5npg](https://user-images.githubusercontent.com/10319195/134905224-f9315225-69a8-43ce-8fd6-4af74aafa224.png)

**Comment by TcNobo** [comment]
Fixed in newest release 2.4: https://github.com/TcNobo/TcNo-osu-Cleaner/releases/tag/2.4
